src/components/CurrentWallet.jsx

```diff
@@ -8,13 +8,19 @@ import DisplayUTXOs from './DisplayUTXOs'
 
 export default function CurrentWallet ({ currentWallet }) {
   const [walletInfo, setWalletInfo] = useState(null)
+  const [redactBalances, setRedactBalances] = useState(!(window.localStorage.getItem('jm-redactBalances') === 'false'))
```

Comment:
> +1 for hiding balances by default. 100

Agreed, let's also rename the property to `showBalances`.

Reply:
Updated. ✅